classes/Group/Group.mjs

```diff
@@ -68,12 +72,18 @@ export default class Group {
                     message: "Invalid roles"
                 }
             }
-            roles = roles.split(" ")
+            roles = roles.toUpperCase().split(" ")
```

Comment:
This only changes them to UPPERCASE when they are passed in a String. `["permission"]` would be unaffected.